source/_components/zabbix.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: "Zabbix"
+description: "Instructions how to integrate Zabbix into Home Assistant."
+date: 2016-12-13 22:57
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: g
+ha_category: 
+featured: false
+ha_release: 0.35
+---
+
+The Zabbix component is the main component to connect to a Zabbix monitoring instance via the Zabbix API.
```

Comment:
Since this the main component I would suggest you add the link to the Zabbix web page as in the Zabbix sensor documentation.